Use four spaces for indetation and follow PSR-1 and PSR-2

merumelu opened the pull request:
- Use four spaces for indetation everywhere.
- Apply PSR-1 and PSR-2 standard to files in `src/` and `api/` by using `php-cs-fixer --level=psr2` plus the following manual fixes:
    - Method names now use camelCase to comply with PSR-1 section 4.3
    - Class files now use the namespace `Godot\AssetLibrary` to follow PSR-1, section 3, and use capitalized file names so they can easier follow both these points from PSR-1, section 1.
        
        >Namespaces and classes MUST follow an "autoloading" PSR: [PSR-0, PSR-4].
        >Class names MUST be declared in StudlyCaps.

Templates were left as is for now (aside from indetation changes) since they might be rewritten anyway.

Everything seems to work fine for me locally but please test. :)

bojidar-bg commented:
Oops, seems like I broke it a bit. Sorry @merumelu

akien-mga commented:
Interesting, GitHub has a web-based conflict resolution editor now... That's not too bad a feature, though it of course introduces a non-cool commit that a proper rebase would have gotten rid of :p

bojidar-bg commented:
Still, I'm getting errors when running this:
```
Message: Call to undefined method Godot\AssetLibrary\Helpers\Utils::errorResponseIfNoUserHasLevel()
File: .../assetlib/src/routes/asset_edit.php
Line: 616
```

bojidar-bg commented on src/routes/asset_edit.php:
```diff
@@ -1,767 +1,847 @@
-  $error = $this->utils->error_reponse_if_not_user_has_level($error, $response, $user, 'moderator', 'You are not authorized to accept this asset edit');
-  if($error) return $response;
+    $error = $this->utils->ensureLoggedIn(false, $response, $body, $user);
+    $error = $this->utils->errorResponseIfNoUserHasLevel($error, $response, $user, 'moderator', 'You are not authorized to accept this asset edit');
```
Problem comes from here..

merumelu commented:
Hello. Sorry for not having time for this until now.

I rebased and fixed the typo in `src/routes/asset_edit.php` mentioned by @bojidar-bg. Hopefully that was the only one, but will retest again later tonight.
